paasta_tools/paasta_maintenance.py

```diff
@@ -221,6 +222,8 @@ def paasta_maintenance():
     elif action == 'up':
         mesos_maintenance.up(hostnames)
     elif action == 'status':
+        ret = "%s" % mesos_maintenance.friendly_status()
```

Comment:
I think at one point, we were using it as a sort of type cast. I think we can drop it now that the functions being called are returning strings. Let me fix this.